soc/xtensa/espressif_esp32/common/Kconfig.soc

```diff
@@ -37,6 +37,14 @@ config ESP_HEAP_SEARCH_ALL_REGIONS
 	  regions. If the region of desired capability is exhausted,
 	  memory will be allocated from other available region.
 
+config SPIRAM_HEAP_SIZE
```

Comment:
Also, add `ESP` or `ESP32` prefix in here.

Reply:
OK, that requires another HAL update pushed here:  https://github.com/zephyrproject-rtos/hal_espressif/pull/235

Reply:
Done, please take another look when you have a chance.